gateway-ha/pom.xml

```diff
@@ -280,23 +326,29 @@
             <version>${dep.jeasy.version}</version>
         </dependency>
 
-        <dependency>
```

Comment:
This dep was added to fix an issue as well .. we need to verify this does not come up again.

Reply:
I see.. you just moved it down to runtime scope.

Reply:
Yep